File: Modul-2/Soal-5/PRAK205-2410817310012-FAISALTANJUNG.c
#include<stdio.h>
#include<math.h>

int main(){
    int a, b;
    float c, tinggi, keliling, luas;
    printf("");
    scanf("%d %d", &a, &b );

    c = ((b * b) - (a * a));
    double alas=sqrt (c);
    tinggi = a;
    keliling = a + b + alas;
    luas = 0.5 * alas * tinggi;

    printf("Alas = %.0f cm \nTinggi = %.0f cm \nKeliling = %.0f cm \nLuas = %.0f cm^2", alas, tinggi, keliling, luas);
    return 0;
}

File: Modul-2/Soal-5/PRAK205-2410817310012-FAISALTANJUNG.py
import math
total_variabel = []
while len(total_variabel) < 2:
    line = input()
    if line :
        total_variabel.extend(map(float, line.split()))

a, b = total_variabel[:2]

c = ((b * b) - (a * a))
alas = math.sqrt(c)

tinggi = a
keliling = a + b + alas
luas = 0.5 * alas * tinggi

print(f"Alas = {alas:.0f} cm\nTinggi = {tinggi:.0f} cm\nKeliling = {keliling:.0f} cm\nLuas = {luas:.0f} cm^2")